sofa-boot-project/sofa-boot-core/rpc-sofa-boot/src/test/java/com/alipay/sofa/rpc/boot/test/SofaBootRpcAllTest.java

```diff
@@ -291,6 +303,26 @@ public void testLoadBalancerAnnotation() throws NoSuchFieldException, IllegalAcc
         Assert.assertTrue("Found roundrobin reference", found);
     }
 
+    @Test
+    public void testConnectionNumAnnotation() throws NoSuchFieldException, IllegalAccessException {
+        Field consumerConfigMapField = ConsumerConfigContainer.class
```

Comment:
It's ok to write this way, but it's a little complicated. It is better to show that both xml and annotation methods can take effect.